src/blockchain-scanner.service.ts

```diff
@@ -92,7 +92,7 @@ export class BlockchainScannerService implements OnApplicationBootstrap {
         }
         const jobs = filteredEvents.map(({ event }) => {
           const { key: jobId, data } = createGraphUpdateJob(event.data.delegatorId, event.data.providerId, UpdateTransitiveGraphs);
-          return this.graphUpdateQueue.add('graphUpdate', data, { jobId });
+          return this.graphUpdateQueue.add(jobId, data, { jobId });
```

Comment:
Why change the `name` (1st param) to be the `jobId`? `name` is meant to describe the type of job, I thought, not the individual job entry.